docs/usage/diff.rst

```diff
@@ -34,3 +34,9 @@ Examples
        +135 B    -252 B file2
     added           0 B file4
     removed         0 B file3
+
+    $ borg diff testrepo::archive1 archive3 --json-lines
```

Comment:
Better use: `borg diff --json-lines testrepo::archive1 archive3`

It usually does not matter / is a matter of preference, but there were weird bugs in python argparse in the past.